ha/core/controllers/pcs/service_controller.py

```diff
@@ -82,3 +99,25 @@ def status(self, service: str, nodeids: list = None) -> dict:
                 status: Succeeded, Failed, InProgress
         """
         raise HAUnimplemented("This operation is not implemented.")
+
+    def clear_resources(self, node_id: str):
+        """
+        Clear resources on node.
+
+        Args:
+            nodeid (str): Private fqdn define in conf store.
+        """
+        try:
+            resources: list = []
+            output, _, _ = self._execute.run_cmd(const.LIST_PCS_RESOURCES, check_error=False)
+            if "NO resources" not in output:
```

Comment:
I think its better to compare string using lower(). Because if that output case changes, this check will break

Reply:
yes, Done